src/DotNetBoilerplate.Application/Reviews/Create/CreateReviewCommand.cs

```diff
@@ -6,5 +6,5 @@ namespace DotNetBoilerplate.Application.Reviews.Create;
 public sealed record CreateReviewCommand(
     Rating Rating,
     string Comment,
-    Guid BookId
+    Guid? BookId = null
```

Comment:
Tutaj zostawmy to nienullowalne. Nie chcemy dopuścić możliwości wejścia jakkolwiek głębiej do nasszego systemu z wartością która nie ma sensu (review nie może istnieć bez książki)